src/server-extension/resolvers/series.ts

```diff
@@ -22,18 +22,24 @@ enum OrderDirection {
   ASC = 'ASC',
 }
 
+type DateRange = '24 HOUR' | '7 DAY' | '14 DAY' | '30 DAY' | '90 DAY' | '180 DAY' | 'ALL DAY'
```

Comment:
Replace with enum for DAY, WEEK, TWO_WEEKS, MONTH, QUARTER, HALF_YEAR

Reply:
Good idea.